backend/src/schema/types/type/EmailAddress.gql

```diff
@@ -19,5 +18,6 @@ type Mutation {
     avatarUpload: Upload
     locationName: String
     about: String
+    termsAndConditionsAgreedVersion: String
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-23T11:25:53Z" title="Friday, August 23rd 2019, 1:25:53 pm +02:00">Aug 23, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -19,5 +18,6 @@ type Mutation {
     avatarUpload: Upload
     locationName: String
     about: String
+    termsAndConditionsAgreedVersion: String!
```


```suggestion
    termsAndConditionsAgreedVersion: String
```
@ogerly I think you can allow this to be `null`. If we import the users, they will have `termsAndConditionsAgreedVersion == null`. So that's a valid situation where this is null. The check in the nuxtjs middleware should still work as expected.